src/main/java/se/munhunger/idp/model/persistant/User.java

```diff
@@ -17,11 +19,44 @@
 @ApiModel(description = "A user")
 public class User {
     @Id
+    @Cascade(CascadeType.MERGE)
     @Column(name = "username", length = 64)
     @ApiModelProperty(value = "The unique identifier for a user")
-    public String username;
+    private String username;
     @ApiModelProperty(value = "A SHA-256 hashed password")
     @Column(name = "password", length = 128)
-    @JsonIgnore
-    public String hashPassword;
+    private String hashPassword;
+
+    public User () {
+
+    }
+
+    public User(String username, String hashPassword) {
+        this.username = username;
+        this.hashPassword = hashPassword;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public String getHashPassword() {
+        return hashPassword;
+    }
+
+    public void setHashPassword(String hashPassword) {
+        this.hashPassword = hashPassword;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
fair enough, men varför?

Reply:
och varför manuellt konvertera den till en json om du vill ha den i json format? använd ett lib för sånt om du vill ha json

Reply:
Hänger inte med vill att jag tar bort getter and setters? Jag skriver dit dem av vana ifall man i framtiden vill processa fälten....